+ CSES Problems (#619)
cpinitiative/usaco-guide: solutions/CSES Finding a Centroid.mdx

---
id: cses-2079
source: CSES
title: Finding a Centroid
author: Dong Liu
---

**Time Complexity**: $\mathcal O(N)$

> For more information about centroids and how to find/use them, see [their module](https://usaco.guide/plat/centroid).

Author: ```suggestion
For more information about centroids and how to find/use them, see [their module](/plat/centroid).
```

---
id: cses-2079
source: CSES
title: Finding a Centroid
author: Dong Liu
---

> # Solution

Reviewer: Also maybe we don't need a heading for the solution here? Idk

> This is a simple Centroid problem where we're asked to find the centroid. For more information about centroids, see [here](https://usaco.guide/plat/centroid).

Reviewer: ```suggestion
For more information about centroids and how to find/use them, see [their module](https://usaco.guide/plat/centroid).
```
I don't think we need to say it's a centroid problem :P
